docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+services:
+  todo-app-dev:
+    container_name: todo-app-dev
+    image: todo-app:dev
+    build:
+      context: .
+      target: development
+      dockerfile: Dockerfile
+    ports:
+      - 5100:5000
+    volumes:
+      - ./todo_app:/todo-app/todo_app 
+    env_file:
+      - ".env"
+
+  todo-app-prod:
+    container_name: todo-app-prod
+    image: todo-app:prod
+    build:
+      context: .
+      target: production
+      dockerfile: Dockerfile
+    ports:
+      - 8080:80
+    env_file:
+      - ".env"
+
+  todo-app-test:
+    container_name: todo-app-test
+    image: todo-app:test
+    build:
```

Comment:
There's nothing _wrong_ with including both image & build, but I do find that you're more likely to confuse people who forget the `--build` flag and keep seeing an old version, and I don't personally particularly benefit from having a specific image name, but open to disagreement there